frontend/src/bundles/common/components/stepper/stepper.tsx

```diff
@@ -24,7 +24,9 @@ type Properties = {
 
 const Stepper: React.FC<Properties> = ({ steps, currentStep, onClickBack }) => {
     const activeStepIndex = steps.indexOf(currentStep);
-    const progressPercent = (activeStepIndex / (steps.length - 1)) * 100;
+    const MAX_PERCENT = 100;
```

Comment:
It is better to declare the constants outside of the component. Not necessary in an external file but outside of the component